src/codemap_details/CSS/WhatToPay.module.css

```diff
@@ -0,0 +1,64 @@
+.head {
+    margin-top: 50px;
+    letter-spacing: 0px;
+    font-size: 2.3rem;
+    font-weight: 390;
+    font-family: Arial;
+    text-align: center;
+    margin-bottom: 50px;
+  }
+  .blue {
+    color: #008dc8;
+  }
+  .jumbo {
```

Comment:
@MiliJolly remove the top padding as its causing a lot of extra space between two components.